### PR TITLE
Display thumbnails in content list

### PR DESCRIPTION
**Pull request by SangeethaTestPress**
- Added icons
- Refactored the content list layouts

### CODE REVIEW

File: course/src/main/java/in/testpress/course/ui/viewholders/BaseContentListItemViewHolder.kt [hunk 1]
```diff
@@ -21,12 +16,12 @@ abstract class BaseContentListItemViewHolder(view: View) : RecyclerView.ViewHold
     private val title: TextView = view.findViewById(R.id.title)
     private val thumbnail: ImageView = view.findViewById(R.id.thumbnail)
     private val scheduledInfo: TextView = view.findViewById(R.id.scheduled_info)
-    private val whiteForeground: View = view.findViewById(R.id.white_foreground)
     private val lockContainer: LinearLayout = view.findViewById(R.id.lock_container)
     val attemptedTickContainer: LinearLayout = view.findViewById(R.id.attempted_tick_container)
     private val lockImage: ImageView = view.findViewById(R.id.lock_image)
     private val scheduledInfoContainer: LinearLayout = view.findViewById(R.id.scheduled_info_container)
     private val contentDetailsContainer: LinearLayout = view.findViewById(R.id.content_details_container)
+    val contentIcon: ImageView = view.findViewById(R.id.content_icon)
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/ui/viewholders/BaseContentListItemViewHolder.kt:24]: content icon means?  It should be contet type icon

File: course/src/main/java/in/testpress/course/ui/viewholders/ContentListItemViewHolder.kt [hunk 2]
```diff
@@ -20,4 +16,13 @@ class ContentListItemViewHolder(view: View) : BaseContentListItemViewHolder(view
...
     }
-}
\ No newline at end of file
+
+    override fun bindContentDetails(content: DomainContent) {
+        super.bindContentDetails(content)
+        when (content.contentTypeEnum) {
+            ContentType.Notes -> contentIcon.setImageResource(R.drawable.writing)
+            ContentType.Attachment -> contentIcon.setImageResource(R.drawable.paperclip)
+            else -> contentIcon.setImageResource(R.drawable.ic_live)
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/ui/viewholders/ContentListItemViewHolder.kt:25]: what about other content types ? video, exam

File: course/src/main/java/in/testpress/course/ui/viewholders/VideoContentListItemViewHolder.kt [hunk 3]
```diff
@@ -46,19 +41,7 @@ class VideoContentListItemViewHolder(view: View) : BaseContentListItemViewHolder
 
     private fun showVideoProgressPercentage(videoWatchedPercentage: Int) {
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/ui/viewholders/VideoContentListItemViewHolder.kt:42]:
```suggestion
    private fun showVideoProgress(videoWatchedPercentage: Int) {
```

**Review by Karthik-0 — CHANGES_REQUESTED** [review]

File: course/src/main/java/in/testpress/course/ui/viewholders/ContentListItemViewHolder.kt (hunk 2 above)
Comment (SangeethaTestPress) [reply course/src/main/java/in/testpress/course/ui/viewholders/ContentListItemViewHolder.kt:25]: For video and exam we have seperate viewholder

**Review by Karthik-0 — APPROVED** [review]

**Pull request merged**